class.jetpack-network.php

```diff
@@ -417,8 +417,12 @@ public function do_subsiteregister( $site_id = null ) {
 		// Save the secrets in the subsite so when the wpcom server does a pingback it
 		// will be able to validate the connection
 		$secrets = $jp->generate_secrets( 'register' );
-		@list( $secret_1, $secret_2, $secret_eol ) = explode( ':', $secrets );
-		if ( empty( $secret_1 ) || empty( $secret_2 ) || empty( $secret_eol ) || $secret_eol < time() ) {
+		if (
+			empty( $secrets[ 'secret_1' ] ) ||
```

Comment:
Consider removing the spaces between the square brackets and single quotes since you are not using variable array indices.